perception/camera_obstacle_detection/yolo_v2_tiny/tensorflow_fp32_coco/README.md

```diff
@@ -0,0 +1,72 @@
+# YOLO V2 Tiny Darknet Convert Tensorflow
+
+This model is converted into tensorflow format directly from the original
+darknet weights from the
+[darknet website](https://pjreddie.com/darknet/yolov2/).
+
+## Provenance
+
+The network definition and weights come from [darknet
+website](https://pjreddie.com/darknet/yolov2/). It is converted to tensorflow
+pb format using [darkflow](https://github.com/thtrieu/darkflow). This
+[script](https://github.com/ARM-software/ML-examples/blob/master/autoware-vision-detector/scripts/get_yolo_tiny_v2.sh)
+was used to produce the final model file.
+
+## Compile and run the example pipeline
+
+All commands should be run from the root of the model zoo directory. First,
+build the TVM docker image by following instructions in the TVM CLI
+[documentation](../../../../scripts/tvm_cli/README.md).
+
+Compile the model by running the TVM CLI script in a docker container.
+
+```bash
+$ export MODEL_DIR=`pwd`/perception/camera_obstacle_detection/yolo_v2_tiny/tensorflow_fp32_coco/
+$ mkdir ${MODEL_DIR}/example_pipeline/build
+$ docker run \
+    -it --rm \
+    -v ${MODEL_DIR}:${MODEL_DIR} -w ${MODEL_DIR} \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    autoware-model-zoo/tvm_cli:local \
+        --config ${MODEL_DIR}/definition.yaml \
+        --output_path ${MODEL_DIR}/example_pipeline/build
```

Comment:
I would prefer if we put these Docker containers in https://hub.docker.com/u/autoware so people know they are Autoware-provided. I'm fine giving you push access to the repo. Are you OK with this?

Reply:
@JWhitleyWork I think that is the best way forward. Thank you.

Reply:
Once they are in, I can push another patch that changes these docs.